packaging/rpm/systemd.spec

```diff
@@ -23,11 +21,7 @@ Summary:        System and Service Manager
 %global github_version %(c=%{version}; echo ${c}|tr '~' '-')
 
 # download tarballs with "spectool -g systemd.spec"
-%if %{defined commit}
-Source0:        https://github.com/packit-service/%{source_name}/archive/%{commit}/%{source_name}-%{shortcommit}.tar.gz
-%else
-Source0:        https://github.com/systemd/systemd/archive/v%{github_version}/%{name}-%{github_version}.tar.gz
-%endif
+Source0:        https://github.com/packit-service/%{name}/archive/%{commit}/%{name}-%{shortcommit}.tar.gz
```

Comment:
I'm not sure if this is going to work, as the `Name` is defined as `systemd`, yet this repo is called `systemd-rhel8-flock` (that's what the original `source_name` shenanigans were for.

Also, after reverting the spec-related commit the`%{commit}` macro has to be renamed back to `%{gitcommit}`.

Reply:
I missed that one.